GameData/RP-0/Tree/ECM-Engines.cfg

```diff
@@ -524,8 +524,8 @@
     XLR25-CW-1 = XLR11-RM-5, 5000
     XLR35-RM-1 = 8000,XLR11-RM-5
     XLR41 = Navaho-PhaseII-TP
-    XLR43-NA-1 = 10000, Navaho-PhaseIII-TP
```

Comment:
Can you explain the reasoning for swapping from "skipping XLR43/NAA75-x saves you 10k" to "XLR43 is free (with the cost shifted to the 110s variant)"?

Reply:
The idea is the XLR43 is Navaho phase III, you could not have the more modern engines without developing it.

I adjusted the costs for it a bit to make more sense, 5000 to Phase III, 5000 to the XLR43-NA-1, and 5000 to the A-6, for the same 15k total cost to get the NAA-75 on top of Navaho. Development of the XLR43 was important, but you could probably save some costs (5k) by not moving to make it a flight ready item. And if it was a flight ready, getting it to run a bit longer for Redstone probably wouldn't be too hard (another 5k).